internal-enrichment/cape-sandbox/src/cape-sandbox.py

```diff
@@ -104,16 +104,25 @@ def _send_knowledge(self, observable, report):
 
         final_observable = self.helper.api.stix_cyber_observable.update_field(
             id=final_observable["id"],
-            input={"key": "hashes.MD5", "value": target["md5"]},
+            input={
+                "key": "hashes",
+                "object_path": "hashes/MD5",
```

Comment:
Miss a `/` ahead

```suggestion
                "object_path": "/hashes/MD5",
```

Example

![image](https://github.com/user-attachments/assets/36d31062-d60a-48bb-9369-81334a0b9123)